hypervisor/src/arch/aarch64/regs.rs

```diff
@@ -216,3 +216,8 @@ arm64_sys_reg!(MPIDR_EL1, 3, 0, 0, 0, 5);
 arm64_sys_reg!(ID_AA64MMFR0_EL1, 3, 0, 0, 7, 0);
 arm64_sys_reg!(TTBR1_EL1, 3, 0, 2, 0, 1);
 arm64_sys_reg!(TCR_EL1, 3, 0, 2, 0, 2);
+
+pub const AARCH64_ARCH_TIMER_PHYS_SECURE_IRQ: u32 = 13;
```

Comment:
nit: What's the motivation for the `AARCH64_ARCH_` prefix - that's implicit in the file. Or is this a common naming convention for this value?

Reply:
I am okay to remove it if it is too verbose.

Reply:
I really don't mind :-)

Reply:
Then I am going to keep it the same way, I guess.